Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1606,8 +1606,23 @@ private bool PrepareFileAccessMonitoringCommon(LoggingContext loggingContext)
             }
 
             // Untrack the globally untracked paths specified in the configuration 
+            var reverseTranslator = m_fileAccessManifest.DirectoryTranslator?.GetReverseTranslator();
             foreach (var path in m_sandboxConfig.GlobalUnsafeUntrackedScopes)
             {
+                // translate the path and untrack the translated one
+                if (reverseTranslator != null)
+                {
+                    var pathString = path.ToString(m_pathTable);
+                    var translatedPathString = reverseTranslator?.Translate(pathString);
+                    var translatedPath = AbsolutePath.Create(m_pathTable, translatedPathString);
+                  
+                    if (pathString != translatedPathString)
+                    {
+                        m_fileAccessManifest.AddScope(translatedPath, mask: m_excludeReportAccessMask, values: FileAccessPolicy.AllowAll | FileAccessPolicy.AllowRealInputTimestamps);
+                        Tracing.Logger.Log.TranslatePathInGlobalUnsafeUntrackedScopes(loggingContext, m_pip.SemiStableHash, m_pip.GetDescription(m_context), pathString, translatedPathString);

```

Comment:
why is only this path logged and not the one 3 lines below?

Reply:
I only log the translation, don't want to spam the log, since this will happen in every pip.